src/lib/utilities/serialized-emoji.ts

```diff
@@ -0,0 +1,60 @@
+import type { IntegerString } from '#lib/common/types';
```

Comment:
Eventually we'll probably want to make @skyra/database-utilities in archid-components for stuff like this